shakenfist/db.py

```diff
@@ -428,6 +425,14 @@ def get_instance_snapshots(instance_uuid):
 
 def add_event(object_type, object_uuid, operation, phase, duration, message):
     t = time.time()
+    LOG.withLabel(object_type, object_uuid).withFields(
```

Comment:
`withLabel` could be subsumed into the `WithFields` call. Maybe something else was intended?